components/browser/toolbar/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,5 @@
     <!-- Content description (not visible, for screen readers etc.): Description for the overflow menu button in the browser toolbar. -->
     <string name="mozac_browser_toolbar_menu_button">Menu</string>
     <string name="mozac_close_button_description">Close</string>
+    <string name="mozac_progress_loading">Loading</string>
```

Comment:
nit: Can you turn this into `mozac_browser_toolbar_progress_loading` ("mozac" + component name + string name)

Reply:
Oh, one more thing: Can you add a comment above this string explaining the context this strings shows up in? Our L10N tools will pick up that comment and show it next to the string to translators. I guess for this string it is worth mentioning that it will be read by talkback.

Reply:
Oops. Apparently I needed to put in a translation for German and Chinese? I grabbed those from Android's core framework.

Reply:
After #2508 that lint check is disabled and will no longer fail the build since it's expected to land code without translations :D